tools/snp.sh

```diff
@@ -578,6 +703,7 @@ build_and_install_amdsev() {
   #build_guest_initrd
 
   # Save binary paths in source file
+  # in RHEL, Will resume to work on adjusting guest path names. 
```

Comment:
Are you done with this? Do you still need this comment?
The issue with comments like this: it's easy to forget to clean them up.